remote/remote.go

```diff
@@ -41,6 +43,13 @@ type options struct {
 	prevImageRepoName   string
 	createdAt           time.Time
 	addEmptyLayerOnSave bool
+	registries          []registry
+}
+
+type registry struct {
+	repository         string
```

Comment:
```suggestion
type reference struct {
	prefix         string
```
I wonder if this is more accurate, since `repository` as used in `getRegistry` could also contain a namespace or tag.